docs/data/toolpad/core/introduction/integration.md

```diff
@@ -290,23 +265,15 @@ To integrate Toolpad Core into your Next.js Pages Router app, follow these steps
 In your root layout file (e.g., `pages/_app.tsx`), wrap your application with the `AppProvider`:
 
 ```tsx title="pages/_app.tsx"
+import * as React from 'react';
 import { AppProvider } from '@toolpad/core/nextjs';
-import { DashboardLayout } from '@toolpad/core/DashboardLayout';
 import { PageContainer } from '@toolpad/core/PageContainer';
+import { DashboardLayout } from '@toolpad/core/DashboardLayout';
 import Head from 'next/head';
 import { AppCacheProvider } from '@mui/material-nextjs/v14-pagesRouter';
 import DashboardIcon from '@mui/icons-material/Dashboard';
 import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
-import { createTheme } from '@mui/material/styles';
-
-export type NextPageWithLayout<P = {}, IP = P> = NextPage<P, IP> & {
-  getLayout?: (page: React.ReactElement) => React.ReactNode;
-  requireAuth?: boolean;
-};
-
-type AppPropsWithLayout = AppProps & {
-  Component: NextPageWithLayout;
-};
+import type { Navigation } from '@toolpad/core';
```

Comment:
Would this be imported from `'@toolpad/core/AppProvider';`?

cc @apedroferreira ?

Reply:
The problem we solve with it doesn't apply to types, but it's less confusing for users to be consistent. It also forces us to organize well.